ngraph/frontend/onnx_import/src/editor/editor.cpp

```diff
@@ -369,3 +403,118 @@ void onnx_import::ONNXModelEditor::set_input_values(
         modify_initializer(*onnx_initializer, name, values, onnx_input);
     }
 }
+
+void onnx_import::ONNXModelEditor::ONNXModelEditor::replace_nodes(
+    std::vector<std::vector<int>> node_indexes, Operator node_generator)
+{
+    std::string new_op_name = "custom_op_" + std::to_string(m_custom_op_ID++);
+
+    shift_node_indexes(node_indexes);
+
+    for (auto node_vector : node_indexes)
+    {
+        auto new_end = std::unique(node_vector.begin(), node_vector.end());
```

Comment:
Is this line just to remove potential duplicates from a single node indices vector? Also, if there are potential duplicates, does it affect the `shift_node_indexes` behavior?

Reply:
good catch, thanks